app/views/static/generic_object/main_custom_button_form.html.haml

```diff
@@ -20,6 +20,40 @@
           = "<#{_('Choose')}>"
       %span.help-block{"ng-show" => "angularForm.button_type.$error.required"}
         = _("Required")
+  .form-group{"ng-if"    => "vm.customButtonModel.button_type == 'ansible_playbook'",
+              "ng-class" => "{'has-error': angularForm.playbook_catalog_item.$invalid}"}
+    #form_playbook_options
```

Comment:
You can probably drop the id, it's not used anywhere,
and it's an empty element in the original template (`app/views/shared/_playbook_options.html.haml`)